www/src/spec.js

```diff
@@ -80,8 +80,10 @@ export function Requirements({ requirements, showSection }) {
       valueGetter(params) {
         return params.row.section.id;
       },
-      sortComparator(v1, v2, row1, row2) {
-        return row1.row.cmp(row2.row);
```

Comment:
We should still use the `cmp` function, just update the logic there. It also might be a good idea to precompute the string we use to sort on since this is executed for basically every pair of requirements.

https://github.com/awslabs/duvet/blob/5e5386879c2d0145c5e0564df59ead54147e8e86/www/src/result.js#L75

Reply:
Thanks for the heads-up. Then I realized the source of the problem was actually the `valueGetter`.